src/components/DynamicFormContainer/DataFileButton/DataFileButton.tsx

```diff
@@ -113,15 +124,17 @@ export const DataFileButton: FunctionComponent<DataFileButton> = ({ onDataFile,
           <FormErrorBanner formErrorTitle="Uploaded data file format has errors." formError={dataFileError} />
         </div>
       )}
-      <div className="p-8 rounded-xl border-dashed border-2" style={{ backgroundColor: "#FFF7E2" }}>
-        <img className="mx-auto mb-8" src={"/upload-icon-dark.png"} />
-        <p className="text-center mb-4">{text.header}</p>
-        <div className="mb-4" data-testid="data-upload-zone" {...getRootProps()}>
-          <input data-testid="config-file-drop-zone" {...getInputProps()} />
-          <Button data-testid="data-upload-button" className="flex mx-auto bg-white text-cerulean hover:bg-cloud-100">
-            {text.buttonText}
-          </Button>
-        </div>
+      <div {...getRootProps()}>
+        <DropZone isDragActive={isDragActive} customDropZoneCss={customDropZoneCss}>
+          <img className="mx-auto mb-8" src={"/upload-icon-dark.png"} />
+          <p className="text-center mb-4">{text.header}</p>
+          <div className="mb-4" data-testid="data-upload-zone" {...getRootProps()}>
+            <input data-testid="config-file-drop-zone" {...getInputProps()} />
```

Comment:
think shouldn't repeatedly use `getRootProps`, not sure what's needs to be done here